src/main/java/net/openhft/chronicle/queue/impl/single/Pretoucher.java

```diff
@@ -36,130 +32,34 @@
  * <p>
  * Resources held by this object will be released when the underlying queue is closed.
  * <p>
- * Alternatively, the {@code shutdown()} method can be called to close the supplied queue and release any other resources. Invocation of the {@code
- * execute()} method after {@code shutdown()} has been called will cause an {@code IllegalStateException} to be thrown.
+ * Alternatively, the {@code close()} method can be called to close the supplied queue and release any other resources. Invocation of the {@code
+ * execute()} method after {@code close()} has been called will cause an {@code InvalidEventHandlerException} to be thrown.
  */
-@Deprecated(/* Move to Queue Enterprise in x.24 */)
+// TODO: x.25 turn into interface
 public class Pretoucher extends AbstractCloseable {
-    static final long PRETOUCHER_PREROLL_TIME_DEFAULT_MS = 2_000L;
-    private final long PRETOUCHER_PREROLL_TIME_MS = Jvm.getLong("SingleChronicleQueueExcerpts.pretoucherPrerollTimeMs", PRETOUCHER_PREROLL_TIME_DEFAULT_MS);
-    private final SingleChronicleQueue queue;
-    private final NewChunkListener chunkListener;
-    private final IntConsumer cycleChangedListener;
-    private final boolean earlyAcquireNextCycle;
-    private final boolean canWrite;
-    private final PretoucherState pretoucherState;
-    private final TimeProvider pretouchTimeProvider;
-    private int currentCycle = Integer.MIN_VALUE;
-    private SingleChronicleQueueStore currentCycleWireStore;
-    private MappedBytes currentCycleMappedBytes;
-
+    @Deprecated(/* remove in x.25 */)
     public Pretoucher(final SingleChronicleQueue queue) {
-        this(queue,
```

Comment:
In these constructors we should probably warn that this is a dummy implementation and that it won't do anything. I didn't even know PretouchUtil existed previously so I wonder how many clients create Pretouchers directly using this class.

Reply:
Have added a warning in static block